lv_conf.h

```diff
@@ -967,7 +975,9 @@
 
 /** FFmpeg library for image decoding and playing videos.
  *  Supports all major image formats so do not enable other image decoder with it. */
+#ifndef LV_USE_FFMPEG
```

Comment:
These will be overwritten if we generate lv_conf.h again from [`lv_conf.defaults`](https://github.com/lvgl/lv_port_pc_eclipse/blob/master/lv_conf.defaults)

It's @Daxidz is working on a better Cmake - lv_conf  interoperability. See here: https://github.com/lvgl/lvgl/pull/7934

Reply:
OK, I remove this change.